### PR TITLE
Mobile previous works

### PR DESCRIPTION
**Pull request by jpol01**
**Portfolio: mobile version**

>>Second part of the Project mobile version.
>>This time we had to add, Previous works and an About me section.
>>Important to mention: Must use the Css- Grid and the Flexbox through this section.

### CODE REVIEW

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @jpol01 👋, 
## Good job so far!🎉
### Summary 👏
- No linter errors ✔
- Descriptive PR message😊
- Good Readme 🚀
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ] I'll implore you always add your commit messages in the present tense and not otherwise🙄 it will be more professional to have `Add the image card 1` to having ``Added the image card 1`😉👍

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -85,14 +85,223 @@ <h4>I am a Software Developer</h4>
...
+        </p>
+
+        <ul class="u-card-technologies">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button class="u-button-card">See Project</button>
+      </div>
+      <div class="about-me-main">
```
[the remark this answers is not in the document]
Comment (jpol01) [reply index.html:243]: I'm sorry I don't really understand what do you mean. The about-me-main container is in a different section already. So, do you want me to just create a space? between lines 242 and 243? thanks. :smile:

[the remark this answers is not in the document]
Comment (jpol01) [reply index.html:246]: Fixed, I moved it out for more semantic code writing

File: index.html [hunk 2]
```diff
@@ -85,14 +85,223 @@ <h4>I am a Software Developer</h4>
...
+            <ul class="about-me-internal-list">
+              <li>CoseKit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </div>
+        </div>
+      </div>
```
[the remark this answers is not in the document]
Comment (jpol01) [reply index.html]: Fixed, I created the li elements and fill them with the corresponding internal title for (Framework, languages and skills) :smile:

**Review by lily-coder — APPROVED** [review]
Hi @jpol01 , 

Thank you for implementing the changes as requested by the previous reviewers. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**